### DISCUSSION TITLE
MBR + IDs profiling

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#346] [Q&A]
The github documentation on DIA-NN MBR contains the following paragraph: 

MBR should be enabled for any quantitative experiment, unless you have a very high quality project-specific spectral library, which you think (i) is likely to provide almost complete coverage of detectable peptides, that is there is no point in trying library-free search + MBR, and (ii) most of the peptides in the library are actually detectable in the DIA experiment. If only (i) is true, might be worth still trying MBR along with Library generation set to IDs profiling.

In my case, only (i) may be true. The library-free searches for subsets ( 96 samples taht are biologically different) of the entire set of raw files ( thousands) do not provide better depth than the library-based search but take significantly longer. Thus the library search is preferred in my case. MBR was active in both cases. So far I have used smart profiling for library generation. 

Q1: What is the difference between the library generation settings? (IDs profiling, IDs, RT & IM profiling, smart profiling, Full profiling)?
Q2: Do I understand correctly that the recommendation to use MBR + IDs profiling relates to the final search to provide quantitative data? (As opposed to a library search for instance)?
Q3:  What are the advantages/disadvantages to be expected when using this setting MBR+IDs profiling? How large would the expected effect size be (e.g. % gain/improvement)?
Q4: Is it correct, that smart profiling is the recommended default setting otherwise?
Q5: Should i also try the 2 other settings?
Q6: I presume these settings improve the library that is generated from the raw files. I further presume that this search-specific library is created from the results of the first pass search and used for the second pass search. Accordingly, the potential improvement of using IDs profiling will only come into effect in the results of the second pass search - correct?

**Comment by vdemichev** [comment]
Q1. Defines which information is updated in the library based on DIA data.
Q2. Sorry, did not get the question, can you please rephrase?
Q3 - Q5 - please just follow the recommendations in the docs. Quantitatively the differences in performance between algorithms will always be highly project-specific.
Q6. Yes, MBR only affects the second pass, the first pass is always equivalent to non-MBR search.

**Comment by wenbostar** [comment]
Hi @vdemichev , could you please explain a bit more about the difference among the different settings of library generation?

**Reply by vdemichev** [reply]
Full profiling extracts empirical spectra for all precursors, smart profiling - only extracts medium- and high- quality empirical spectra, while for precursors with low quality spectra the original library spectrum is retained (i.e. if searching with in silico lib that would be the predicted spectrum). IDs profiling mode does not extract any empirical info from the data, but just filters the list of precursors to those confidently identified (at a global q-value filter).

Best,
Vadim

**Reply by wenbostar** [reply]
Hi Vadim,

Thanks a lot. This is very helpful. For smart profiling (the default option), when extracting empirical spectra (i.e., DIA spectra) to generate a new library, both RTs and matched fragment ion intensities extracted from the DIA spectra are used to replace the RTs and fragment ion intensities from the original spectral library, right? Are all matched fragment ions used?

Bo

**Reply by vdemichev** [reply]
In version 1.9, IDs, RT and IM profiling is default, in general that's what is recommended.
Yes, RTs aligned to the reference scale.
No, not all fragment ions used and only for those precursors which have OK quality empirical spectra.

**Reply by wenbostar** [reply]
So in version 1.9 using "the IDs, RT and IM profiling" setting, to generate a new library, DIA-NN filters the list of precursors to those confidently identified (at a global q-value filter), and the fragment ion intensities in the new library are directly from the original library but the RTs are aligned to the reference scale, right?

**Reply by vdemichev** [reply]
Yes, exactly

**Reply by wenbostar** [reply]
Thanks a lot.
